### PR TITLE
Fix dependency issues

### PR DESCRIPTION
**Pull request by barzilaydn**
- Move clojure/lang to aphyr/riemann/client to prevent jar hell in parent projects also depending on clojure.
- Unused yammer metrics-core was removed (same artifact name as _used_ codahale mertics-core)
- Minor update (0.2.13)

### CODE REVIEW

**Comment by itaifrenkel** [comment]
@barzilaydn Didn't look at it yet... tests fail. why ?

File: pom.xml [hunk 1]
```diff
@@ -1,180 +1,174 @@
...
-    <repository>
-      <id>clojars</id>
-      <name>Clojars</name>
-      <url>https://clojars.org/repo</url>
-    </repository>
-  </distributionManagement>
-</project>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.forter</groupId>
```
Comment (itaifrenkel) [inline pom.xml]: com.forter.com.aphyr

**Comment by itaifrenkel** [comment]
@barzilaydn Why have you deleted yamr metrics? And how are we supposed to upgrade to the latest version from upstream project, if we just deleted files from it?

**Comment by barzilaydn** [comment]
Look at the CI output [https://travis-ci.org/forter/riemann-java-client/builds/98706049] - It's seems like there is no travis.yml so it doesn't know what tests to run. (It tries to use `rake`, it thinks it's ruby for some reason)

**Comment by itaifrenkel** [comment]
@barzilaydn I see that upstream separated yamer metrics. Have you proposed your clojure change as a PR?

**Pull request closed without merging**